tests/integration/conftest.py

```diff
@@ -1,6 +1,8 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+# pylint: disable=missing-module-docstring,missing-function-docstring
```

Comment:
For `missing-module-docstring`, it could just be `"""Fixtures for integration tests"""`

Reply:
I spotted 3 functions without a docstring, is it worth just adding a docstring for them?

Reply:
Fixed, thanks!